docs/docs/developing/database_schema.md

```diff
@@ -17,3 +17,15 @@ To visualize the schema we've used
 to modify it and save it back as xml in the repo.
 
 ![](../../images/database_schema.jpg)
+
+# How to add a new field
+Rough draft of how to add a new field
```

Comment:
The ideas are correct, but before it can be merged, I'd prefer if instead of bullet points alone there was some redaction that will explain each one, to make it more pleasant to read.